src/Core/Core.hs

```diff
@@ -42,15 +42,16 @@ data AnnTerm b a
   | AnnMatch b (AnnAtom b a) [AnnArm b a] -- ^ Pattern matching
 
   | AnnExtend b (AnnAtom b a) [(Text, Type a, AnnAtom b a)] -- ^ Record extension
+  | AnnTupleUn b [AnnAtom b a] -- ^ Unboxed tuple
```

Comment:
I don't like the name.

Reply:
Enable MagicHash everywhere and call it `AnnTuple#`?

Reply:
(That's not a serious suggestion but we really need a better name)

Reply:
(That's not a serious suggestion but we really need a better name)

Reply:
I originally had `AnnUnboxedTup`, but it felt a wee bit too verbose. Dunno.

Reply:
Maybe `AnnValues`?